packages/components/stories/data-table/data-table.stories.css

```diff
@@ -0,0 +1,7 @@
+.test__table {
+    height: calc(100% - 42px);
+    flex: 1;
+}
+.test__row {
```

Comment:
what are these tests for?

Reply:
I mean, can we call them something else?

Reply:
Sure we can. Just a class to apply some styles.